Tests/js/views/assets/ez-actionbarview-tests.js

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright (C) eZ Systems AS. All rights reserved.
+ * For full copyright and license information view LICENSE file distributed with this source code.
+ */
+YUI.add('ez-actionbarview-tests', function (Y) {
+    var Assert = Y.Assert,
+        viewTest;
+
+    viewTest = new Y.Test.Case({
+        name: "eZ Action Bar View test",
+
+        setUp: function () {
+            this.contentType = {};
+
+            Y.eZ.CreateContentActionView = Y.Base.create('createContentActionView', Y.eZ.ButtonActionView, [Y.eZ.Expandable], {}, {
+                contentTypeGroups: {},
+                contentTypeSelectorView: {},
+                contentType: {}
+            });
+
+            this.view = new Y.eZ.ActionBarView({
+                contentType: this.contentType
+            });
+        },
+
+        tearDown: function () {
+            delete Y.eZ.CreateContentActionView;
+            this.view.destroy();
+        },
+
+        "Should instantiate createContentActionView with contentType": function () {
+            for (var i = 0; i < this.view.get('actionsList').length; i++){
+                if (this.view.get('actionsList')[i].get('actionId') === 'createContent'){
```

Comment:
there's a small issue here, if the `createContent` action is not found the test should fail.